lib/eventboss/runner.rb

```diff
@@ -25,6 +25,7 @@ def launch
         Eventboss::DevelopmentMode.setup_infrastructure(queues) if config.development_mode?
 
         begin
+          validate_client!(client, config)
```

Comment:
@TanyaAkulich How about moving this validation one or two levels up, eg. to the [`CLI.run`](https://github.com/AirHelp/eventboss/blob/master/lib/eventboss/cli.rb#L32). 

If the config is not good, there's no point starting the runner, launcher and all.

Reply:
You're right. I don't have a strong opinion about moving it in hierarchy. 

But if you decide to keep this validation in the `Runner`, maybe you could do it before starting the `Launcher`? There's no point in *launching* anything if we know that it's not configured properly and it will fail.

Reply:
don't get it 
the validation happened before `launcher.start` 🤔

Reply:
Yes, but the launcher is instantiated with incorrect client setup.